bitrise.yml

```diff
@@ -61,10 +73,14 @@ workflows:
     - install-missing-android-tools@3:
```

Comment:
We can remove this step now, modern Android Gradle Plugin versions download the missing NDK version and other dependencies automatically.

Reply:
We also don't want to give the impression that Bitrise has "missing Android tools" 😅 The most common versions are preinstalled, and Gradle can install any missing version at build time.